### PR TITLE
Gamelab respect debuggerDisabled

### PR DESCRIPTION
**Pull request by cpirich**
* Gamelab now honors the `debuggerDisabled` level flag to determine when the debugger is disabled. All new levels created will have the debugger enabled by default. To simplify the transition, we've disabled the debugger in all of the existing Gamelab curriculum levels and level builders can go back and selectively choose to enable it where it makes sense.
* Fixed a long-standing debugger bug where the last statement remains highlighted in droplet/ace after resuming the debugger (e.g. pressing "Continue" after being stopped at a breakpoint)
* Added debugger highlighting tests that verify proper calls to ace's `addMarker` and `removeMarker`

### CODE REVIEW

File: apps/src/gamelab/GameLab.js [hunk 1]
```diff
@@ -230,9 +230,7 @@ GameLab.prototype.init = function (config) {
   // able to turn them on.
   config.noInstructionsWhenCollapsed = true;
 
-  // NOTE: We will go back to using !config.level.debuggerDisabled soon,
-  // but are testing with project levels only for now
-  var breakpointsEnabled = config.level.isProjectLevel;
+  var breakpointsEnabled = !config.level.debuggerDisabled;
```
Comment (caleybrock) [inline apps/src/gamelab/GameLab.js:233]: This files looks good.

File: apps/src/lib/tools/jsinterpreter/JSInterpreter.js [hunk 2]
```diff
@@ -645,10 +645,14 @@ export default class JSInterpreter {
         return;
       }
     }
-    if (reachedBreak && atMaxSpeed) {
-      // If we were running atMaxSpeed and just reached a breakpoint, the
-      // code may not be selected in the editor, so do it now:
-      this.selectCurrentCode();
+    if (atMaxSpeed) {
```
Comment (caleybrock) [inline apps/src/lib/tools/jsinterpreter/JSInterpreter.js:648]: I'd like to see the highlighting bug behavior tested, but code looks good.

**Comment by caleybrock** [comment]
The preferred method of updating the levels would be to run a script on the LB machine since this can cause conflicts, but it looks like there aren't any Game Lab levels being edited on LB right now, so I think this could work.

**Comment by caleybrock** [comment]
Looks like circle has a JS interpretor failure, related to this change?

**Comment by cpirich** [comment]
Trying to re-build on CircleCI - the last run hit this error inside `appLabgameLabTest.log`:
```
29 01 2018 18:34:52.419:ERROR [launcher]: PhantomJS crashed.
```

**Pull request merged**